tests/Dockerfile.fedora

```diff
@@ -72,11 +72,6 @@ RUN if which dnf5; then \
 	# Fedora 41+ \
 	rpm -e --nodeps --nodb \
 	    rpm-plugin-ima; \
-    else \
```

Comment:
Nah, I think we can just drop the whole "if dnf 5 exists" conditional here. Its only purpose was for differentiating between F40 and F41 :smile:

Reply:
Oh, indeed

Reply:
Dropped.